binding.gyp

```diff
@@ -1,7 +1,7 @@
 {
     'targets':[ {
         'target_name':'jetmon',
-        'cflags_cc': [ '-fexceptions','-O3' ],
+        'cflags_cc': [ '-fexceptions','-O3', '-Wno-unused-result' ],
```

Comment:
This is required to suppress the warning caused by the call [`cb_func->Call( isolate->GetCurrentContext(), isolate->GetCurrentContext()->Global(), 3, argv );`](https://github.com/Automattic/jetmon/blob/88c62724abcb83bab1c7b46b664d1054fa3dff3e/src/main.cpp#L36) as `cb_func->Call` will try to return a result, but we don't use it.